invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -64,6 +66,23 @@ def apply_idx(affiliation):
     }
 
 
+def localize_vocabulary_list(field_name, key, attr, obj):
+    """Localize vocabulary."""
+    field_data = attr.get("metadata", {}).get(field_name)
+
+    if not field_data:
+        return None
+
+    localized = []
+    for item in field_data:
+        localized.append({
+            "id": item.get("id"),
+            "title": item.get("metadata", {}).get("title", {}).get("en")
```

Comment:
This is hardcoded, should be changed once agreed.

Reply:
This can use the `current_i18n.locale` as [here](https://github.com/inveniosoftware/invenio-rdm-records/pull/319/files#diff-60d2e985baca9c1e2ddd61f92355612c6707af6901d2242e8d473a726af5cf30R188) no?

Reply:
Yes, was just waiting for the green light on the overall implementation. But yes :)